README.md

```diff
@@ -1,64 +1,111 @@
 
-## Architecture of the app
+## Use manual off-chain group locally
 
-![Bandada-Semaphore Off-chain App Architecture](https://github.com/vplasencia/bandada-semaphore-demo/assets/52170174/b8d43564-01b3-4b7c-ae56-6efb1c5a8773)
+**Please make sure you are on the `main` branch.**
 
-## Run Locally
+- [NodeJS](https://nodejs.org/en) >= v18.17.0
+- A [supabase](https://supabase.com/) free tier project.
+- A local copy of [Bandada](https://github.com/privacy-scaling-explorations/bandada)
 
-### Clone the Repository
+Clone the Bandada repository and follow the [README](https://github.com/privacy-scaling-explorations/bandada/blob/main/README.md) to run it locally and create your first manual off-chain group:
+
+```bash
+git clone https://github.com/privacy-scaling-explorations/bandada.git
+```
+
+To get started, create a [Supabase account](https://supabase.com/dashboard/sign-up) and a free-tier project with basic configuration.
+
+Once your project is ready, access the `Table Editor` from your project dashboard (you can use the [Supabase CLI](https://supabase.com/docs/guides/cli/local-development) if you prefer) and create the following tables with the columns as shown in the image:
+
+- `feedback`, which will store all feedback (= signals) sent from users (= identities).
+- Store all nullifier hashes in the `nullifier_hash` table to prevent double signaling. Refer to the [Semaphore documentation](https://docs.semaphore.pse.dev/glossary#nullifier) for more information.
+- Store all Semaphore group roots in the `root_history` table to fix the [Merkle Tree](https://github.com/semaphore-protocol/semaphore/issues/98) (= [Semaphore groups](https://github.com/semaphore-protocol/semaphore/issues/98)) roots expiration issue. Refer to the [conversation](https://github.com/semaphore-protocol/semaphore/issues/98) on GitHub for more information.
+
+![Tables schema](https://github.com/vplasencia/bandada-semaphore-demo/assets/20580910/e6c4362f-8f50-4ed2-87a1-a624a9b1052c)
+
+## Use credential off-chain group locally
+
+**Please make sure you are on the `credential-group-app` branch.**
+
+You must meet the same prerequisites and follow the same steps as previously outlined for the manual off-chain group but instead you have to create a credential off-chain group.
+
+**nb.** This tutorial assumes you are using Github Followers (_MinFollowers_) as credential provider.
+
+You need to **[Register a new OAuth application](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app)** using your Github account. Then configure the `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` environment variables in the [Bandada APIs](https://github.com/privacy-scaling-explorations/bandada/blob/main/apps/api/.env.example) with the values that Github gives you after setting up the OAuth application. Note that this is necessary for Bandada to be able to act as a credential provider and validator for your group.
```

Comment:
Did you test this part? Because I remember that GitHub needed `https` to check the credentials, then I didn't test these things locally.

Reply:
This should work fine. We can leave as is right now or flag as experimental. What do you prefer?